packages/browser/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "extends": "../../tsconfig.json",
```

Comment:
Is this giving us any value extending the root tsconfig.json? It looks like we redefine everything in `@tsconfig/node14` except `skipLibCheck` and `forceConsistentCasingInFileNames`, and ultimately this build is for browsers right? (Or am I misunderstanding?)

Reply:
I had that thought -- I think having a root "tsconfig" to extend from is super common and potentially quite useful, so I have this as a kind of placeholder.

Reply:
Added a more sensible tsconfig.